applications/google_rise_2016.md

```diff
@@ -0,0 +1,459 @@
+recruitment strategy for additional program staff and/or volunteers.**
+
+Zach is currently the only full-time staff for Hack Club. There are a number of
+community members that are very active that Zach works closely with and will
+continue to during 2016.
+
+We plan to recruit staff through the networks of our supporters, friends, and
+club leaders. Volunteers are recruited through our existing club leader base.
+
+**Why is this project needed in this community?**
+
+There is a dearth of organizations in high schools that support computer science
+education. Since Hack Club is the first-to-market for high school coding clubs,
+we have a very unique opportunity to build the organization that we always
+wished existed -- one that is built and sustained by students, for students.
+
+### Impact
+
+Please provide more information on who will benefit from the computer science
+education project if you receive funding from the RISE Awards.
+
+**Local town (specify)**
+
+N/A - National/international project
+
+**Country (specify)**
+
+Focused on the USA, but we are also starting international clubs
+
+**Region (specify)**
+
+Focused on North America, but we are international
+
+**Projected total number of youth served by this project**
+
+Other organizations that follow our model (DECA, FBLA, Key Club, etc) typically
+reach between 100,000 and 300,000 students each year.
+
+These organizations only have a presence in the USA. Since Hack Club is
+international, we hope to eventually expand our reach to the millions.
+
+**Projected demographics of the youth reached by this project**
+
+_Please use approximate percentages to list the demographics of the youth that
+your proposed project will serve this year, such as age ranges, race/ethnicity,
+gender etc. For example, 25% boys, 75% girls._
+
+We project our demographics in 2016 to mirror those in 2015, though we do plan
+on serving more young women. Our demographics in 2015 are as follows:
+
+- 32% women (compare with 20% female AP CS test takers)
+- 37% black/Hispanic (compare with 13% black/Hispanic AP CS test takers in USA)
+
+**What are the anticipated results (objectives/outputs) for this project? (list
+up to 5)**
+
+Our goal is to turn members of Hack Clubs into hackers: people who create what
+they see missing in the world and aren't afraid to break the rules. We'll know
+this is happening if:
+
+- Members are continuing to write code outside of Hack Club
+- Members are working on long-term projects and publishing them
+- Members are winning hackathons and other competitions
+- Members are hosting their own events (hackathons, meetups, etc)
+
+**How do you plan to measure and evaluate your results?**
+
+We'll continue to measure our primary KPIs: the number of clubs and the stories
+we're hearing from club leaders and members.
+
+Additionally, we'll begin measuring attendance of Hack Clubs week-to-week and
+their attrition rates.
```

Comment:
Maybe, we don't really have concrete steps to start doing this though. I say keep it simple for now.